neo/SmartContract/Native/NativeContract.cs

```diff
@@ -92,7 +91,6 @@ internal bool Invoke(ApplicationEngine engine)
             VMArray args = (VMArray)engine.CurrentContext.EvaluationStack.Pop();
             if (!methods.TryGetValue(operation, out ContractMethodMetadata method))
                 return false;
-            if (!method.AllowedTriggers.HasFlag(engine.Trigger)) return false;
```

Comment:
Before `RegisterValidator` was only for `Application`, now for `Verification` too?

Reply:
`System.Contract.Call` is disabled from `Verification`. So there's no way to call any native contract in `Verification` trigger.